### DISCUSSION TITLE
Periodic Table Typo - Fluorine

### DISCUSSION THREAD

**Discussion by DoctorDizzee** [rocksdanister/lively#1368] [General]
In the interactive periotic table Fluorine is accidently spelled Fourine.

**Comment by rocksdanister** [comment]
You can edit it, close wallpaper and rightclick wallpaper in library -> Open File location -> open index.html in notepad.

**Reply by DoctorDizzee** [reply]
Thank you for telling me about this. I will do so. Have a great day.
